README.md

```diff
@@ -32,7 +31,7 @@ If compared to saving websites with `wget -mpk`, this tool embeds all assets as
     $ snap install monolith
 
 ## Usage
-    $ monolith https://lyrics.github.io/db/p/portishead/dummy/roads/ -o portishead-roads-lyrics.html
+    $ monolith https://lyrics.github.io/db/P/Portishead/Dummy/Roads/ -o portishead-roads-lyrics.html
```

Comment:
what happened with the previous url?

Reply:
That website was modified to preserve casing in directory and file names, no longer lowercasing paths (akin to how Wikipedia does it).